src/web/components/Card/Card.tsx

```diff
@@ -99,6 +102,35 @@ const starWrapper = css`
 	margin-top: 2px;
 `;
 
+const labTitleOrNot = (format: Format) => {
+	if (format.theme === Special.Labs) {
+		return css`
+			${textSans.medium({ fontWeight: 'bold' })}
+		`;
+	}
+	return css`
+		${headline.xxxsmall({ fontWeight: 'regular' })}
+		line-height: 20px;
+	`;
+};
```

Comment:
I think this is already set in `CardHeadline`?

Reply:
It's not set as `textsans` just `headline` in `CardHeadline`